docs/source/using/addons.adoc

```diff
@@ -38,7 +38,8 @@ oc adm policy add-scc-to-group anyuid system:authenticated
 
 [NOTE]
 ====
-Comment lines, starting with the '#' character, can be inserted anywhere in the file.
+- Comment lines, starting with the '#' character, can be inserted anywhere in the file.
+- Commands start with `!` character  makes execution idempotent i.e. a command can be executed multiple times without changing the final behavior of the add-on.
```

Comment:
`Command starting with !...`

Reply:
@praveenkumar  I did not want you to remove the information about it ignores the command failure.  I should have been more clear. Sorry for the confusion 
```
Commands starting with `!` character  ignores the execution failure. With the help of this command execution in add-on will be idempotent i.e. a command can be executed multiple times without changing the final behavior of the add-on.
```